src/stories/containers/transparency-report/transparency-comments/comment-item.tsx

```diff
@@ -16,9 +16,9 @@ const CommentItem = ({ comment, code }: Props) => {
   return (
     <CommentItemContainer isLight={isLight}>
       <UTCDate isLight={isLight}>
-        {`Comment on ${DateTime.fromISO(comment.commentDate ?? '')
+        {`Comment on ${DateTime.fromISO(comment.timestamp ?? '')
```

Comment:
I can see that `timestamp` is a required field on `CommentsBudgetStatementDto` and you are expecting an empty value here, if this value can be empty the field in the DTO should be set as optional to avoid possible errors in other places where it can be used. Otherwise, the null check here is not needed. Do we have that info from the models in the API?

Reply:
The edge case could be that there's some internal server error...
In this case, no comment will be created
So provably right now is not necessary if the API work as comment about, but just to prevent